Mastodon/Scene/SuggestionAccount/SuggestionAccountViewController.swift

```diff
@@ -168,46 +98,37 @@ extension SuggestionAccountViewController: UITableViewDelegate {
             )
         }
     }
+
+    func tableView(_ tableView: UITableView, viewForFooterInSection section: Int) -> UIView? {
+        guard let footerView = tableView.dequeueReusableHeaderFooterView(withIdentifier: SuggestionAccountTableViewFooter.reuseIdentifier) as? SuggestionAccountTableViewFooter else {
+            return nil
+        }
+
+        footerView.delegate = self
+        return footerView
+    }
 }
 
 // MARK: - AuthContextProvider
 extension SuggestionAccountViewController: AuthContextProvider {
     var authContext: AuthContext { viewModel.authContext }
 }
 
+// MARK: - UserTableViewCellDelegate
+extension SuggestionAccountViewController: UserTableViewCellDelegate {}
+
 // MARK: - SuggestionAccountTableViewCellDelegate
-extension SuggestionAccountViewController: SuggestionAccountTableViewCellDelegate {
-    func suggestionAccountTableViewCell(
-        _ cell: SuggestionAccountTableViewCell,
-        friendshipDidPressed button: UIButton
-    ) {
-        guard let tableViewDiffableDataSource = viewModel.tableViewDiffableDataSource else { return }
-        guard let indexPath = tableView.indexPath(for: cell) else { return }
-        guard let item = tableViewDiffableDataSource.itemIdentifier(for: indexPath) else { return }
-        
-        switch item {
-        case .account(let user):
-            Task { @MainActor in
-                cell.startAnimating()
-                do {
-                    try await DataSourceFacade.responseToUserFollowAction(
-                        dependency: self,
-                        user: user
-                    )
-                } catch {
-                    // do noting
-                }
-                cell.stopAnimating()
-            }   // end Task
+extension SuggestionAccountViewController: SuggestionAccountTableViewCellDelegate { }
+
+
+extension SuggestionAccountViewController: SuggestionAccountTableViewFooterDelegate {
+    func followAll(_ footerView: SuggestionAccountTableViewFooter) {
+        viewModel.followAllSuggestedAccounts(self) {
+            OperationQueue.main.addOperation {
```

Comment:
💡  No need to change, but I think we're mainly using GCD directly 👇 
```suggestion
            DispatchQueue.main.async {
```

Reply:
Great, that you saw that. I'm the only one using `OperationQueue.main.addOperation`. I changed there in the other places, too 👍